.changeset/rich-apes-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Add docs about config option to auto clean up orphans
```

Comment:
You actually don't need a changeset for this type of change as you are not changing anything in the package you listed 👍

Reply:
Thank you, I was not sure about this, but I saw another PR where you actually mentioned that it is required to trigger a website rebuild. When is it then required? Should I remove it in this case, or leave it and you suggestion is more for future cases?

Reply:
The [guidance](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets) is that you don't need a changeset for documentation changes with the exception to that being `README` files as those also show up on NPM. A changeset will tell the release workflow that a package has a change and needs a new version to be built and published to NPM. In this case your change is not in a package (these are found in the `/packages` and `/plugins` folders) therefore it does not need a changeset. Also, the changesets are used to build up the `CHANGELOG` and release notes having this note in the log for the `@backstage/plugin-catalog` would be confusing, well to me at least :)

Now for the changes you made, seeing as they are in the `/docs` folder, as soon as it gets merged it will trigger a rebuild of backstage.io

Finally, any chance you can share where I was giving contradictory comments? As a reviewer I like to keep myself consitent, seeing that would help me understand where I went wrong and help me improve 👍

Reply:
Yes, I would remove it, as I mentioned seeing that in the change log would be confusing

Thanks for sharing where I made this comment before. I think I'll open a PR to update the guidance on this to mention the `README` to make it a bit more clear

Reply:
I have removed the changeset, and updated the suggested config